README.md file of Website update (#73)
SaptarshiSarkar12/Drifty: Website/README.md

# Website

> **In simple words, we can describe a README file as a guide that gives users a detailed description of a project you have worked on.**

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> **It can also be described as documentation with guidelines on how to use a project. 

Reviewer: ```suggestion
**A README file can be described as documentation with guidelines on how to use a project. 
```

> Usually it will have instructions on how to install and run the project but we're already worked on it which is the main README file our project.**

Reviewer: ```suggestion
Usually it will have instructions on how to download, install, run and contribute to the project but we've already worked on it which is the main [README](https://github.com/SaptarshiSarkar12/Drifty/blob/master/README.md) file of our project.**
```

> - This section contain information of website like how's your website look like, you can see instructions videos, 

Reviewer: ```suggestion
- This section contain information of website like how the [website of this project](https://saptarshisarkar12.github.io/Drifty/) look like, you can see instructions videos, 
```

> layers section and lots of images that are used to make website and lot more options

Reviewer: ```suggestion
layers section and lots of images that are used to make website and lot more options.
```

> - and this website section also contain files like html, css, and js by which you can make necessary changes in this project

Reviewer: ```suggestion
- and this website section also contain files like HTML, CSS, and JavaScript by which you can make necessary changes in this project's website.

```

# Website
**A README file can be described as documentation with guidelines on how to use a project. 

> Usually it will have instructions on how to download, install, run and contribute to the project but we've already worked on it which is the main [README](https://github.com/SaptarshiSarkar12/Drifty/blob/master/README.md) file of our project.**

Reviewer: ```suggestion
Usually it will have instructions on how to download, install, run and contribute to the project but we've already worked on it which is the main [README](https://github.com/SaptarshiSarkar12/Drifty/blob/master/README.md) file of our [project](https://github.com/SaptarshiSarkar12/Drifty).**

```

- This section contain information of website like how the [website of this project](https://saptarshisarkar12.github.io/Drifty/) look like, you can see instructions videos, 

> layers section and lots of images that are used to make website and lot more options.

Reviewer: ```suggestion
and lots of images (located in [Resources](https://github.com/SaptarshiSarkar12/Drifty/tree/master/Website/Resources) folder) that are used to make the website.

```

> - and this website section also contain files like HTML, CSS, and JavaScript by which you can make necessary changes in this project's website.

Reviewer: ```suggestion
- this website folder also contains files like HTML, CSS, and JavaScript by which you can make necessary changes in this project's website.
```
